relay.go

```diff
@@ -281,6 +282,22 @@ func (r *Relayer) Relay(f *Frame) (shouldRelease bool, _ error) {
 	return r.handleCallReq(cr)
 }
 
+func (r *Relayer) updateCallReqContinueChecksum(f *Frame, cs Checksum) {
+	rbuf := typed.NewReadBuffer(f.SizedPayload())
+	rbuf.SkipBytes(1) // flags
+	rbuf.SkipBytes(1) // checksum type
```

Comment:
do you want to note that this should match the current checksum's type? don't think the check in the code is necessary, but the note may be useful